packages/flutter/test/widgets/grid_view_test.dart

```diff
@@ -536,4 +536,31 @@ void main() {
     expect(tester.getTopLeft(find.byKey(target)), const Offset(600.0, 0.0));
     expect(tester.getBottomRight(find.byKey(target)), const Offset(800.0, 200.0));
   });
+
+  testWidgets('GridView crossAxisSpacing', (WidgetTester tester) async {
```

Comment:
Can you please add the following comment after this line:

```suggestion
  testWidgets('GridView crossAxisSpacing', (WidgetTester tester) async {
    // Regression test for https://github.com/flutter/flutter/issues/27151.